cub/cub/agent/agent_merge_sort.cuh

```diff
@@ -79,7 +81,7 @@ struct AgentBlockSort
   // Types and constants
   //---------------------------------------------------------------------
 
-  static constexpr bool KEYS_ONLY = std::is_same<ValueT, NullType>::value;
+  static constexpr bool KEYS_ONLY = ::cuda::std::is_same<ValueT, NullType>::value;
```

Comment:
We are requiring C++17 nowadays, so we can be more efficient here and below:
```suggestion
  static constexpr bool KEYS_ONLY = ::cuda::std::is_same_v<ValueT, NullType>;
```

Reply:
Applies to more places below.